libs/sdk-model/src/execution/buckets/index.ts

```diff
@@ -137,8 +177,30 @@ export function newBucket(
                 } as one of the items for bucket ${localId}.` +
                     "Please make sure that you are not trying to put an array of items into a bucket that only accepts single item.",
             );
+        } else if (typeof i === "object") {
+            if (Object.keys(i).indexOf("Default") > -1) {
+                const identifiers = getAttributeDisplayFormIdentifiers(i).map((identifier) => {
+                    const k = Object.keys(identifier)[0];
+                    const value = identifier[k];
+                    return `${k}: ${value}`;
+                });
+                invariant(
+                    false,
+                    `${contentErrorMessage} It looks like you used an attribute from generated metadata containing more than one display forms. Use one of the following display forms instead: ${identifiers}.`,
+                );
+            }
+            const keys = intersection(AGGREGATION_KEYS, Object.keys(i));
+            if (!isEmpty(keys)) {
+                const identifier = getIdentifier(i[keys[0]]);
+
+                invariant(
+                    false,
+                    `${contentErrorMessage} It looks like you used an object ${identifier} from generated metadata. You need to use one of the following aggregation functions instead: ${keys}.`,
```

Comment:
It would help readability of the result if you used
```js
`...following aggregation functions instead: ${keys.join(", ")}.`
```
The same applies for line 189 and the `${identifiers}` token.